ops/ansible/roles/bfd-pipeline/templates/bluebutton-data-pipeline.sh.j2

```diff
@@ -15,6 +15,12 @@ export NEW_RELIC_METRIC_KEY='{{ data_pipeline_new_relic_metric_key }}'
 export RDA_JOB_ENABLED='{{ data_pipeline_rda_job_enabled }}'
 export RDA_JOB_INTERVAL_SECONDS='{{ data_pipeline_rda_job_interval_seconds }}'
 export RDA_JOB_BATCH_SIZE='{{ data_pipeline_rda_job_batch_size }}'
+{% if data_pipeline_rda_job_starting_fiss_seq_num is defined %}
```

Comment:
is the if block necessary, what will the functionality be if the SEQ_NUM variables are set to the empty string?